README.md

```diff
@@ -1 +1,151 @@
-# React_Capstone_Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 React Capstone <a name="about-project"></a>
+
+**React Capstone** was built to inform different kinds of cuisine.You can find many cuisines and recipe
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML ,CSS ,JavaScript and React</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Different kinds of cuisines**
+- **Detailed recipes**
+- **Navigating to detail**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://react-capstone-project-wiu2.onrender.com/)
```

Comment:
### **Feedback on Live Version Deployment: 🌐**

Hi @MayPyone,

I hope you’re having a productive and pleasant day! I noticed the project has been deployed and the link has been well placed in the README, as per the project requirements, which is great! 🌟 However, upon visiting the link, it seems to be returning a "Bad Gateway" error. This usually implies there might be issues with the deployment configuration or the server hosting your application. 

### **Action Required: 🛠️**
1. **Review Deployment Configuration:** Please revisit your deployment settings on Render, and ensure that all configurations are set correctly, including any environment variables that your project might be using.
   
2. **Check Console Logs:** If available, review any error logs provided by the hosting service, they usually provide insights into what might be going wrong.

3. **Re-Deploy:** Once you’ve made any necessary adjustments, redeploy the application and ensure that both the home and detail page work as expected.

4. **Update README Link:** If changes in configuration result in a new URL, please remember to update the link in the README accordingly.

### **Suggestions: 🌿** [OPTIONAL]
- After rectifying the error, you might want to test the deployment link in different browsers to ensure cross-browser compatibility.
- Ensure that your deployed version is up to date with the latest codebase, and all features are working as they should be.

I know resolving deployment issues can sometimes be challenging, but I'm confident you'll manage to get it working flawlessly! 💪 Looking forward to seeing your project live! 🚀

Once the issues are resolved, please don’t hesitate to tag me or drop a comment in the PR thread, and I will review it promptly. 

Happy Coding! 🌟

Best Regards,
Temitope